theory/mathematics/geometry.md

```diff
@@ -0,0 +1,35 @@
+# Rotation matrix {docsify-ignore}
+
+Any 3 dimensional coordinate system (including conventional Cartesian axes x, y and z) can be rotated to another 3 dimensional coordinate system as long as they maintain the orthogonality among the axes. In linear algebra, this is called change of basis. 
+
+Thus any properties described by a vector in 3 dimensional axes could be rotated in a new set of axes too with the equation below.
```

Comment:
Thus any `property` described `as` a vector in three-dimensional axes could be rotated `to` a new set of axes`:`